server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -1323,7 +1315,7 @@ private VolumeVO orchestrateResizeVolume(long volumeId, long currentSize, long n
             return volume;
 
         } catch (Exception e) {
-            throw new CloudRuntimeException("Exception caught during resize volume operation of volume UUID: " + volume.getUuid(), e);
+            throw new CloudRuntimeException(String.format("Failed to resize volume operation of volume UUID: [%s] due to - %s", volume.getUuid(), e));
```

Comment:
@slavkap I think @nvazquez meant you keep `e.getMessage()` there but also pass exception object as second param of the `CloudRuntimeException` as it has `CloudRuntimeException(String message, Throwable th)`
```suggestion
            throw new CloudRuntimeException(String.format("Failed to resize volume operation of volume UUID: [%s] due to - %s", volume.getUuid(), e.getMessage()), e);
```

Reply:
@shwstppr, thanks! I've misunderstood the comment (sometimes I'm reading selectively... sorry about that)

Reply:
Sorry if I wasn't clear @slavkap  - yes, I meant what @shwstppr explained - thanks for the fix

Reply:
NP, @nvazquez, I didn't read your suggestion correctly. Thanks all for testing and code review :)